js/src/ui/TxHash/txHash.js

```diff
@@ -121,10 +124,21 @@ class TxHash extends Component {
           max={ maxConfirmations }
           value={ value }
           color='white'
-          mode='determinate'
-        />
+          mode='determinate' />
         <div className={ styles.progressinfo }>
-          <abbr title={ `block #${blockNumber.toFormat(0)}` }>{ count } { unit }</abbr>
+          <abbr title={ `block #${blockNumber.toFormat(0)}` }>
+            {
+              value === 1
+                ? <FormattedMessage
+                  id='ui.txHash.confirmation'
+                  defaultMessage='{count} confirmation'
+                  values={ { count } } />
+                : <FormattedMessage
+                  id='ui.txHash.confirmations'
+                  defaultMessage='{count} confirmations'
+                  values={ { count } } />
```

Comment:
There is native support in react-intl, honestly I cannot quite remember how it works, but will take a look and update accordingly.